### PR TITLE
Collection class

### PR DESCRIPTION
**Pull request by gabichuelas**
@rrabinovitch @sciencefixion @evilaspaas1 
Check out the collection class :)

### CODE REVIEW

**Review by sciencefixion — APPROVED** [review]
Looks great! Chop up that stat_tracker!

File: lib/collection.rb [hunk 1]
```diff
@@ -1,5 +1,28 @@
-module TeamStats
+require_relative './readable'
+require_relative './game'
+require_relative './team'
+require_relative './game_team'
 
+class Collection
```
Comment (rrabinovitch) [inline lib/collection.rb]: looooove it - this looks great. thanks for spearheading this creative solution to refactoring!
do we also have a `collection_test` in the works as well? let me know if you'd like support with that piece!

Comment (rrabinovitch) [reply lib/collection.rb]: (I know most of the test file content will be related to the methods that get transferred into the `collection` file, but just want to make sure we're not forgetting the testing piece in this process)

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -335,12 +315,9 @@ def fewest_tackles(season)
   end
 
   # TEAM STATISTICS
-  # Uses helper methods from TeamStats module
 
   def team_info(id)
-    teams.find do |team|
-      team.team_id == id
-    end.info
+    find_team_by_id(id).info
```
Comment (rrabinovitch) [inline lib/stat_tracker.rb:320]: nice combo with these methods!

**Review by rrabinovitch — APPROVED** [review]
sweet! let's just be sure to be building out a `statistics_test` file as we move forward with this refactor

**Pull request merged**